src/components/ActionBar/ActionBar.tsx

```diff
@@ -1,15 +1,10 @@
-/* eslint-disable jsx-a11y/no-static-element-interactions */
-/* eslint-disable jsx-a11y/click-events-have-key-events */
 import * as React from 'react';
-import { useId } from 'react';
-import cn from 'classnames';
+import { useId, useState, forwardRef } from 'react';
 
 import type { ClickHandler } from './Context';
 import { ActionBarContext } from './Context';
-import classes from './ActionBar.module.css';
-import { ActionBarIcon } from './ActionBarIcon';
-import { ActionBarActions } from './ActionBarActions';
 import { ActionBarContent } from './ActionBarContent';
+import { ActionBarHeader } from './ActionBarHeader';
 
 export interface ActionBarProps {
```

Comment:
Pick headingLevel prop from HeadingProps in FDS so we don't have to update this everytime if they update and all other props that exist elsewhere. 
```suggestion
export interface ActionBarProps extends Pick<HeadingProps, 'headingLevel' {
```